plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -137,15 +145,36 @@ class NetworkingFormSection extends mixin(StoreMixin) {
       address = `${hostname}${Networking.L4LB_ADDRESS}${port}`;
     }
 
+    const loadBalancerTooltipContent = (
```

Comment:
Yeah, I think we are expecting to roll tooltips out further and those will definitely want to be centralised. Is it worth spending some time now trying to fix this, or shall we press on with what we've got? @mlunoe , what's your feeling?